packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -48,7 +49,20 @@ export const HelpCenterContactPage: FC = () => {
 		email
 	);
 	const { data: supportAvailability } = useSupportAvailability( 'CHAT' );
-	const isLoading = renderChat.isLoading || renderEmail.isLoading || isLoadingTickets;
+	const featureEnabledGPTResponse = config.isEnabled( 'help/gpt-response' );
+	const [ isLoadingExperimentAssignment, experimentAssignment ] = useExperiment(
+		'caplyso_helpcenter_gpt_answer_contact_deflection',
+		{ isEligible: featureEnabledGPTResponse }
+	);
+
+	const loadGpt =
+		! isLoadingExperimentAssignment && experimentAssignment?.variationName === 'treatment';
```

Comment:
I'd wrap this with a `Boolean()`, it may return `undefined` on network failure; when `isLoadingExperimentAssignment` is false and `experimentAssignment` is `undefined`.